### PR TITLE
Refactor/cleanup

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ ] Feature
- [X] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Simply clean up unnecessary comments and put all interface files into the proper folder.

### What ticket(s) do the changes resolve?

Please Link Issues to this PR as necessary.
#51 

### Notes for the Reviewer
This is just a quick clean up, no major changes/features have been added.

### How to Test Changes?
n/a

### Next Iterations
Have other project members go through this process as well

### CODE REVIEW

File: src/components/Planet/Planet.tsx [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
 import { Link } from 'react-router-dom';
+import { PlanetProps } from '../../interface';
 import './planet.css';
 
-interface PlanetProps {
-  name: string,
-  id: number,
-  planetFact: string
-}
-
```
Comment (alexthompson207) [inline src/components/Planet/Planet.tsx:10]: Good job moving this, makes it more readiable!

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 2]
```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import './planetInfo.css';
-import { PlanetBio } from '../../interface';
-
-interface InfoProps {
-  currentPlanet: PlanetBio;
-}
```
Comment (alexthompson207) [inline src/components/PlanetInfo/PlanetInfo.tsx:8]: Same here!

File: src/components/SortBox/SortBox.tsx [hunk 3]
```diff
@@ -1,32 +1,29 @@
 import React from 'react';
+import { Props } from '../../interface';
 import './SortBox.css';
 
-interface Props {
-  updateSort: (event: React.MouseEvent<HTMLInputElement>) => void
-}
-
```
Comment (alexthompson207) [inline src/components/SortBox/SortBox.tsx:7]: And here!

**Review by alexthompson207 — APPROVED** [review]
Nice work cleaning up the code! Exporting the interfaces makes for more readable components!

**Review by knees4bees — APPROVED** [review]
Good cleanings.

**Pull request merged**